src/server/pfs/server/driver_v2.go

```diff
@@ -62,37 +63,57 @@ func newDriverV2(env *serviceenv.ServiceEnv, txnEnv *txnenv.TransactionEnv, etcd
 	if err != nil {
 		return nil, err
 	}
-	gcClient, err := gc.NewClient(db)
-	if err != nil {
-		return nil, err
-	}
+	tracker := tracker.NewPGTracker(db)
 	chunkStorageOpts, err := env.ChunkStorageOptions()
 	if err != nil {
 		return nil, err
 	}
-	chunkStorageOpts = append([]chunk.StorageOption{chunk.WithGarbageCollection(gcClient)}, chunkStorageOpts...)
-	d2.storage = fileset.NewStorage(objClient, chunk.NewStorage(objClient, chunkStorageOpts...), env.FileSetStorageOptions()...)
+	chunkStorage := chunk.NewStorage(objClient, chunk.NewPGStore(db), tracker, chunkStorageOpts...)
+	d2.storage = fileset.NewStorage(fileset.NewPGStore(db), tracker, chunkStorage, env.FileSetStorageOptions()...)
 	d2.compactionQueue, err = work.NewTaskQueue(context.Background(), d2.etcdClient, d2.prefix, storageTaskNamespace)
 	if err != nil {
 		return nil, err
 	}
-	go d2.master(env, objClient, db)
+	go d2.master(env)
 	go d2.compactionWorker()
 	return d2, nil
 }
 
-func newDB() (*gorm.DB, error) {
+func newDB() (db *sqlx.DB, retErr error) {
+	defer func() {
+		if db != nil {
+			db.MustExec(`DROP SCHEMA IF EXISTS storage CASCADE`)
+			fileset.PGStoreApplySchema(db)
```

Comment:
Minor nit, but can we us the word `Create` rather than `Apply`. That lines up more with the Postgres operation name.

Reply:
It might not just be create at some point, it could include other migrations. What about `SetupPostgresStore(db)` ?
This will likely change as the database migration stuff is fleshed out.